packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -493,8 +537,12 @@ class FlutterPlugin implements Plugin<Project> {
             String maxPluginNdkVersion = projectNdkVersion
             int numProcessedPlugins = getPluginList().size()
 
-            getPluginList().each { plugin ->
-                Project pluginProject = project.rootProject.findProject(plugin.key)
+            getPluginList().each { pluginObject ->
+                assert pluginObject.name instanceof String
```

Comment:
Can you link to some evidence that pluginObject.name will always be a string?

Reply:
Evidence is linked in `shared.groovy`.